tests/recorder_pause_test.cc

```diff
@@ -0,0 +1,69 @@
+#include "cris/core/msg_recorder/record_file.h"
+
+#include "gtest/gtest.h"
+
+#include <unistd.h>
+
+#include <chrono>
+#include <cstring>
+#include <filesystem>
+#include <memory>
+#include <string>
+#include <thread>
+#include <utility>
+
+namespace cris::core {
+
+class RecorderPauseTest : public testing::Test {
+   public:
+    void SetUp() override { std::filesystem::create_directories(test_temp_dir_); }
+
+    void TearDown() override { std::filesystem::remove_all(test_temp_dir_); }
+
+    std::filesystem::path test_temp_dir_{
+        std::filesystem::temp_directory_path() / (std::string("CRPauseTestTmpDir.") + std::to_string(getpid()))};
+
+    const std::string exchange_symbol_str = std::string("TEST_HUOBI") + "__" + std::string("TEST_PERPETUAL");
```

Comment:
Why huobi？Why do you care about "symbols"?

Those are all internal informations and **DO NOT** put them in the public repos.